.github/workflows/test-lint.yml

```diff
@@ -22,10 +22,11 @@ jobs:
         with:
           PATTERNS: |
             **/*.go
-            */go.mod
-            */go.sum
+            **/go.mod
+            **/go.sum
       - uses: golangci/golangci-lint-action@v3
         with:
           version: v1.52.1
           args: --timeout 10m
           github-token: ${{ secrets.github_token }}
+          working-directory: hermes
```

Comment:
There is #5 issue to fix the linter for all plugins. Fixing it might require using a matrix or a script to discover the plugins. Maybe defining a `go.mod` at the root of the project would help, but it doesn't make sense there.